### PR TITLE
Re-former

### PR DESCRIPTION
**Pull request by keneogu**
This project is part of Microverse program, designed to learn the basics of Ruby on Rails creating a Re-former app where you can create Users by using forms.
This project has the following features:

Users Table to create and store users in the database.

Forms to create new users.

Forms to edit existing users.

### CODE REVIEW

**Review by NtwaliHeritier — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUESTED🛑🛑🛑🛑🛑**
Hello team,
Great work on this project. It's impressive the way you covered almost everything👏👏
Let's make some few changes before getting it approved

[REQUIRED]: Kindly add the update method in the controller
[REQUIRED]: When pushing to git, always remember to add node_modules to gitignore
[REQUIRED]: In your html form, in the action attribute use "users_path" instead of "/users"
[REQUIRED]: you didn't use form_tag. Kindly use it before using form_with
[OPTIONAL]: In your routes config, specify the root_path, so that you don't know to manually specify it in the url

That's it.
giphy (2)
Make the required changes and ask for another review👏👌
Happy coding💻

**Review by Georjane — APPROVED** [review]
## Status: PR Approved 💯 🥇  
### Hi there Team ✋🏽 
Congratulations on successfully completing the forms project. You have resolved all required issues by the previous Code reviewer. You can go ahead and merge this PR
_Feel free to reach out to me on Slack if you have any questions, Witah Georjane_ 🤝 
Good luck on your project and Happy Coding 💻 

**Witah Georjane
Code Reviewer at Microverse Inc
[portfolio](https://witahgeorjane.tech/) | [linkedIn](https://www.linkedin.com/in/witah-georjane/)**

**Pull request merged**
